app/gui/view/src/project.rs

```diff
@@ -55,9 +95,8 @@ ensogl::define_endpoints! {
     }
 
     Output {
-        searcher_opened                     (NodeId),
+        searcher_opened                     (SearcherInput),
         adding_new_node                     (bool),
-        searcher_input                      (Option<NodeId>),
```

Comment:
`searcher_opened` wasn't used before at all, so I decided to replace usages of `searcher_input` with it.